pandas/core/generic.py

```diff
@@ -10315,10 +10315,6 @@ def _doc_parms(cls):
 level : int or level name, default None
     If the axis is a MultiIndex (hierarchical), count along a
     particular level, collapsing into a %(name1)s.
-numeric_only : bool, default None
```

Comment:
Removing this will remove `numeric_only` for all other statistical function that actually accept `numeric_only`. We only want to remove this for `mad`

Reply:
Makes sense...I shall correct my mistake. Thank you for guiding me @mroeschke

Reply:
Please review this commit!